libs/shared/ui/src/lib/components/slider/slider.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import { render } from '__tests__/utils/setup-jest'
+import { fireEvent } from '@testing-library/react'
+import { Slider, SliderProps } from './slider'
+
+let props: SliderProps
+
+let container
+
+beforeEach(() => {
+  props = {
+    min: 10,
+    max: 100,
+    step: 10,
+  }
+
+  container = document.createElement('div')
+  document.body.appendChild(container)
+})
+describe('Slider', () => {
+  it('should render successfully', () => {
+    const { baseElement } = render(<Slider {...props} />)
+    expect(baseElement).toBeTruthy()
+  })
+
+  it('should apply class name', () => {
+    props.className = 'some-class-name'
+
+    const rendered = render(<Slider {...props} />)
+
+    const slider = rendered.container.querySelector('.slider')
+
+    expect(slider?.classList.contains('some-class-name')).toBe(true)
+  })
+
+  it('should have label', () => {
```

Comment:
description should be more explicit because as it is the text seems to test what the framework is already doing for us